pkgs/development/python-modules/sphinx-click/default.nix

```diff
@@ -0,0 +1,47 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  sphinxHook,
+  # Build system
+  setuptools-scm,
+  # Dependencies
+  click,
+  docutils,
+  pbr,
+  sphinx,
+}:
+
+buildPythonPackage rec {
+  pname = "sphinx-click";
+  version = "6.0.0";
+  pyproject = true;
+
+  build-system = [
+    setuptools-scm
+  ];
+
+  nativeBuildInputs = [
+    sphinxHook
+  ];
+
+  dependencies = [
+    click
+    docutils
+    pbr
```

Comment:
`pbr` should go to `build-system`.
https://github.com/click-contrib/sphinx-click/blob/7f29172b10d4dc251af414ae01e38c3b970686a1/setup.py#L6